src/components/Error.tsx

```
@@ -0,0 +1,26 @@
+import { Ionicons } from '@expo/vector-icons';
+import { Text, TouchableOpacity, View } from 'react-native';
+import { ErrorProps } from '../types';
+
+export default function Error({ onPressRetry, text }: ErrorProps) {
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)